app/gateway/db-less-mode.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "DB-less mode"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+tags:
+    - deployment-topologies
+
+no_version: true
+
+breadcrumbs:
+  - /gateway/
+
+description: "Explains how {{site.base_gateway}} can be run without a database using only in-memory storage for entities."
+
+related_resources:
+  - text: Deployment topologies
+    url: /gateway/deployment-topologies/
+  - text: "{{site.base_gateway}} deployment models"
+    url: /gateway/deployment-models/
+---
+
+{{site.base_gateway}} can be run without a database using only in-memory storage for entities. 
+We call this DB-less mode. When running {{site.base_gateway}} DB-less, the configuration of 
+entities is done in a second configuration file, in YAML or JSON, using declarative configuration.
+
+The combination of DB-less mode and declarative configuration has a number
+of benefits:
+
+* Reduced number of dependencies: No need to manage a database installation
+  if the entire setup for your use-cases fits in memory.
+* Automation in CI/CD scenarios: Configuration for
+  entities can be kept in a single source of truth managed via a Git
+  repository.
+* Enables more deployment options for {{site.base_gateway}}.
+
+{:.warning}
+> **Important**: [decK](/deck/) also manages configuration declaratively, but it requires
+a database to perform any of its sync, dump, or similar operations. Therefore, decK 
+can't be used in DB-less mode.
+
+<!--vale off -->
+{% mermaid %}
+flowchart TD
+
+A(<img src="/assets/logos/kogo-white.svg" style="max-height:20px" class="no-image-expand"/> {{site.base_gateway}} instance)
+B(<img src="/assets/logos/kogo-white.svg" style="max-height:20px" class="no-image-expand"/> {{site.base_gateway}} instance)
+C(<img src="/assets/logos/kogo-white.svg" style="max-height:20px" class="no-image-expand"/> {{site.base_gateway}} instance)
```

Comment:
I don't know if you can do anything about this, but the logo is barely visible in light mode

Reply:
I'll change it to the blue one